Jetcaster/wear/src/main/java/com/example/jetcaster/WearApp.kt

```diff
@@ -101,17 +104,27 @@ fun WearApp() {
                 ) {
                     LatestEpisodesScreen(
                         playlistName = stringResource(id = R.string.latest_episodes),
-                        onShuffleButtonClick = {
-                            // navController.navigateToPlayer(it[0].episode.uri)
-                        },
+                        // TODO implement change speed
+                        onChangeSpeedButtonClick = {},
                         onPlayButtonClick = {
                             navController.navigateToPlayer()
                         }
                     )
                 }
                 composable(route = YourPodcasts.navRoute) {
                     PodcastsScreen(
-                        onPodcastsItemClick = { navController.navigateToPlayer() },
+                        onPodcastsItemClick = { navController.navigateToPodcastDetails(it.uri) },
+                        onErrorDialogCancelClick = { navController.popBackStack() }
+                    )
+                }
+                composable(route = PodcastDetails.navRoute) {
+                    PodcastDetailsScreen(
+                        // TODO implement change speed
+                        onChangeSpeedButtonClick = {},
+                        onPlayButtonClick = {
+                            navController.navigateToPlayer()
```

Comment:
Should this start playback also?

Reply:
Same for next line?

Reply:
At the moment when pressing on Play 2 things happen:
- navigate to Player
- `onPlayEpisode(episodes[0].toPlayerEpisode())` which is a method on the View model to set the episode and starts playing: 
```
        episodePlayer.currentEpisode = episode
        episodePlayer.play()
     ```